src/ansys/fluent/core/services/field_data.py

```diff
@@ -922,6 +954,88 @@ def _extract_field(field_datatype, field_size, chunk_iterator):
         return fields_data
 
 
+@dataclass
+class Node:
+    """Node class for mesh.
+
+    Attributes:
+    -----------
+    x : float
+        x-coordinate of the node.
+    y : float
+        y-coordinate of the node.
+    z : float
+        z-coordinate of the node.
+    """
+
+    _id: int
+    x: float
+    y: float
+    z: float
+
+
+class CellElementType(Enum):
+    """Element types for a cell element."""
+
+    # 3 nodes, 3 faces
+    TRIANGLE = 1
+    # 4 nodes, 4 faces
+    TETRAHEDRON = 2
+    # 4 nodes, 4 faces
+    QUADRILATERAL = 3
+    # 8 nodes, 6 faces
+    HEXAHEDRON = 4
+    # 5 nodes, 5 faces
+    PYRAMID = 5
+    # 6 nodes, 5 faces
+    WEDGE = 6
+    # Arbitrary number of nodes and faces
+    POLYHEDRON = 7
+    # 2 nodes, 1 face (only in 2D)
+    GHOST = 8
+    # 10 nodes, 4 faces
+    QUADRATIC_TETRAHEDRON = 9
+    # 20 nodes, 6 faces
+    QUADRATIC_HEXAHEDRON = 10
+    # 13 nodes, 5 faces
+    QUADRATIC_PYRAMID = 11
+    # 15 nodes, 5 faces
+    QUADRATIC_WEDGE = 12
+
+
+@dataclass
+class Element:
+    """Element class for mesh.
+
+    Attributes:
+    -----------
+    element_type : CellElementType
+        Element type of the element.
+    node_indices : list[int]
+        0-based node indices of the element.
+    """
+
+    _id: int
+    element_type: CellElementType
+    node_indices: list[int]
```

Comment:
Connectivity for polyhedral cells is not yet properly supported. This class will be bit more complex when we support that in future.